src/utils.js

```diff
@@ -210,12 +213,14 @@ var Utils = exports.Utils = {
    **/
   disablePrivacyBadgerForOrigin: function(origin){
     if(localStorage.disabledSites === undefined){
-      localStorage.disabledSites = JSON.stringify([origin]);
+      var obj = {};
+      obj[origin] = true;
+      localStorage.disabledSites = JSON.stringify(obj);
```

Comment:
You should never access `localStorage` naked. `setItem` and `getItem` are wrappers for this and they are the recommended way of doing. This is an excellent SO answer http://stackoverflow.com/questions/12632463/is-localstorage-getitemitem-better-than-localstorage-item-or-localstoragei/24904863#24904863.